Symfony/CS/Tests/AbstractIntegrationTest.php

```diff
@@ -36,16 +36,16 @@
  * --REQUIREMENTS--
  * php=5.4**
  * hhvm=false***
- * --INPUT--
- * Code to fix
  * --EXPECT--
- * Expected code after fixing*****
+ * Expected code after fixing
+ * --INPUT--
+ * Code to fix*****
  *
  *     * Additional fixers may be omitted.
  *    ** PHP minimum version. Default to current running php version (no effect).
  *   *** HHVM compliant flag. Default to true. Set to false to skip test under HHVM.
  *  **** Black listed filters may be omitted.
- * ***** When the expected block is omitted the input is expected not to
+ * ***** When the input block is omitted the expected part is expected not to
```

Comment:
good point.

"When the input block is omitted only the expected parts is checked to not be changed by the fixers."

Ok? Other proposal?